### PR TITLE
using html and css

### PR DESCRIPTION
**Pull request by maanikandan**
### This PR contains :
-Header
-Content
-Social links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,178 @@
...
+
+- I would like to thank Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="License"></a>
+
+This project is [MIT](./License) licensed.
```
Comment (AbelG101) [inline README.md]: - [ ] Kindly fix this MIT link as it's currently broken. This is a licensed project so please make sure it works. 🎯

File: README.md [hunk 2]
```diff
@@ -1 +1,178 @@
...
+- **Social-Links**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](N/A)
```
Comment (AbelG101) [inline README.md]: - [ ] Please make sure the live demo link either works or just provide a "coming soon" text. ✔

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      src="https://kit.fontawesome.com/b1fa342dbf.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <header>
+      <div class="header">
+        <a href="#" class="brand">Manikandan</a>
+        <i class="fa-sharp fa-solid fa-bars"></i>
```
Comment (AbelG101) [inline index.html:20]: - [ ] Kindly add this menu icon inside a nav element for semantics as it ensures screen readers and other supportive devices understand what it is 👍.

File: LICENSE
Comment (AbelG101) [inline LICENSE:1]: **_THIS COMMENT IS REGARDING THE BRANCH, PR TITLE, AND DESCRIPTION_** 
- [ ] Please make sure you have a descriptive and concise branch name as it'll help you distinguish from the different features later on when you start adding multiple features. ✔
    - [ ] The `feature-branch`  branch name isn't descriptive enough, as you'll have different features in your project.
- [ ] Also kindly make sure you add a descriptive PR title for each milestone as it helps you easily figure out what's included.
   - [ ] Please make sure your PR description is a  short summary describing the introduced changes in more detail.

**Review by AbelG101 — CHANGES_REQUESTED** [review]
## Hi @maanikandan 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔

## Required Changes ♻️
- _Check the comments under the review._
- It's a really great start to your portfolio 💯 , you just have a few changes to make and you're on your way 🏇🏾 .

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — COMMENTED** [review]
Hi @maanikandan,

terminator

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a Pr with a meaningful title, and the live demo link hasn't been replaced properly with the "Coming soon..." phrase. Please just replace this `[Live Demo Link]("coming soon")` with this `Coming soon...`

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <header>
+      <div class="header">
+        <nav>
+          <a href="#" class="brand">Manikandan</a>
+        <i class="fa-sharp fa-solid fa-bars"></i>
```
Comment (franclobo) [inline index.html:20]: - [ ] As requested by the previous CR just the hamburger button should be inside the `nav` tag for the best practices of HTML & CSS.

**Review by franclobo — COMMENTED** [review]
Hi @maanikandan, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @maanikandan 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Use of semantic HTML ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**